packages/cfd/src/Components/__tests__/cfd-personal-details-form.spec.js

```diff
@@ -141,20 +136,31 @@ describe('<CFDPersonalDetailsForm />', () => {
         expect(screen.queryByTestId('dt_cfd_details_form_description')).not.toBeInTheDocument();
     });
 
-    it('should disable Citizenship and Tax residence fields if they were submitted earlier & is_fully_authenticated is true', () => {
+    it('should disable Citizenship and Tax residence fields if they were submitted earlier & immutable from BE', () => {
         const values = {
             citizen: 'Indonesia',
             tax_residence: 'Indonesia',
             tax_identification_number: '',
             account_opening_reason: '',
+            place_of_birth: 'Indonesia',
         };
-        render(<CFDPersonalDetailsForm {...props} is_fully_authenticated value={values} />);
+
+        const changeable_fields = ['tax_identification_number'];
+        render(
+            <CFDPersonalDetailsForm
+                {...props}
+                is_fully_authenticated
+                value={values}
+                changeable_fields={changeable_fields}
+            />
+        );
 
         expect(screen.getByRole('textbox', { name: /citizenship/i })).toBeDisabled();
         expect(screen.getByRole('textbox', { name: /tax residence/i })).toBeDisabled();
+        expect(screen.getByRole('textbox', { name: /place of birth/i })).toBeDisabled();
         expect(screen.getByRole('textbox', { name: /tax identification number/i })).toBeEnabled();
         expect(screen.getByText(/account opening reason/i)).toBeEnabled();
-        expect(screen.getByRole('button', { name: /next/i })).toBeEnabled();
+        expect(screen.getByRole('button', { name: /next/i })).toBeDisabled();
```

Comment:
Can this test case be split into 2?  1 for checking if the field where values entered are disabled and 2nd for texting the reason for disable of button due to validation failure